gradle/ext.gradle

```diff
@@ -4,13 +4,13 @@ enableFeaturePreview("GRADLE_METADATA")
 // set some global variables
 gradle.ext {
     kotlin_version = kotlin_version
-    kotlinx_coroutines_version = '1.3.9-native-mt-2'
+    kotlinx_coroutines_version = '1.4.2'
```

Comment:
needs to add `-native-mt`

Reply:
I understand, but it will not solve my problem on iOS using firebase-kotlint-sdk for example:
```   > Could not resolve org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.2.```

Reply:
we can switch it over to `1.4.2-native-mt` as well

Reply:
Yes, but it's edge case when we're able to :) Will do

Reply:
another way is to exclude the sub dependency when including it, something like

```groovy
compile('firebase-kotlin-sdk') {
     exclude 'org.jetbrains.kotlinx:kotlinx-coroutines-core'
  }
```

Reply:
not sure what you mean by edge case. Using `-native-mt` (for the main artifact) is the Jetbrains recommended way if you want to use iOS.

Reply:
Okay